### ISSUE TITLE
3 thing to improve with map editor

### ISSUE THREAD

**Issue by houk001** [cedbossneo/openmower-gui#50]
1. Functionality of moving whole area move to additional icon when pressed this action is possible, i move whole area couple of time when I want to move one dot.
2. When  saving map.bag create backup of old map.bag.  One copy is enough. 
3. Ability to select several dots and move them with alignment.

**Comment by fallingcats** [comment]
> 1. Functionality of moving whole area move to additional icon when pressed this action is possible, i move whole area couple of time when I want to move one dot.

Agreed.

> 2. When  saving map.bag create backup of old map.bag.  One copy is enough.

`openmower-gui` doesn't directly save to the map (in fact it doesn't even have access to it), it only sends commands to modify the map to the ros map service

> 3. Ability to select several dots and move them with alignment. 

If I understand that correctly, that's already possible with the latest version. Click on an area twice, hold shift and drag a selection box. To update, run `sudo podman auto-update`

**Comment by houk001** [comment]
Shift is working, thank you. Next is select couple of dots and alignment this dots by the line of first and last dot selected, this will be cool option to correct map after recording area.

**Comment by fallingcats** [comment]
> Shift is working, thank you. Next is select couple of dots and alignment this dots by the line of first and last dot selected, this will be cool option to correct map after recording area.

If you just want a straight line you can also go ahead and just delete all selected points

**Comment by houk001** [comment]
Yes, i know but deleted points cause mower to drive fast and i have some part of outline that i want to drive slow, because of bumps or driving very close to items.

**Comment by houk001** [comment]
I have mower running slow when i record drive slow. After deletion 200 dots mower drive this outline area fast. Maybe points was slightly not in line.

**Comment by Jahnkeanater** [comment]
I can't get the merge tool to save a map at all. It says saving failed then corrupts the map file.

**Comment by cedbossneo** [comment]
Fixed in the v2 branch — whole-area dragging now requires holding Ctrl (or Cmd on macOS). This prevents accidental area moves when trying to select or move individual points. The cursor also changes to indicate when area-drag is available.
